packages/stacked_firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.13
+- Remove signInWithPhoneNumber (implements Web Platform)
```

Comment:
This has not been removed? Can we update the changelog to match the latest hanges please